src/chrome/content/rules/Python.org.xml

```diff
@@ -1,68 +1,76 @@
-		* empleo.es.python.org
-		* forum.pl.python.org
-		* hg.es.python.org (incomplete certificate chain)
-		* lists.es.python.org (incomplete certificate chain)
-		* mail.pl.python.org
-		* redesign.python.org
+	HSTS preloaded:
+		- docs
+		- doc
+		- hg
 
-	No working URL known:
-		* front.python.org
+	Incomplete certificate chain:
+		- hg.es
+		- lists.es
 
-	Refused:
-		* buildbot.python.org
-		* dinsdale.python.org
-		* www.es.python.org
-		* ftp.python.org
-		* legacy.python.org
-
-	Secure connection failed:
-		* blog.python.org
-		* blog-cn.python.org
-		* blog-de.python.org
-		* blog-es.python.org
-		* blog-fr.python.org
-		* blog-ja.python.org
-		* blog-ko.python.org
-		* blog-pt.python.org
-		* blog-ro.python.org
-		* blog-ru.python.org
-		* blog-tw.python.org
+	Mismatched:
+		- calendario.es
+		- documentos-asociacion.es
+		- empleo.es
+		- www.pl
+		- forum.pl
+		- ns1.pl
+		- redesign
+		- uk
 
+	TLS protocol errors:
+		- blog
+		- blog-cn
+		- blog-de
+		- blog-es
+		- blog-fr
+		- blog-ja
+		- blog-ko
+		- blog-pt
+		- blog-ro
+		- blog-ru
+		- blog-tw
+		- dinsdale
+		- legacy
 -->
-<ruleset name="Python.org (partial)">
 
+<ruleset name="Python.org">
 	<target host="python.org" />
+	<target host="www.python.org" />
+	<target host="africa.python.org" />
 	<target host="bugs.python.org" />
 	<target host="cheeseshop.python.org" />
+	<target host="devguide.python.org" />
+	<target host="console.python.org" />
 	<target host="discuss.python.org" />
+	<target host="es.python.org" />
+	<target host="www.es.python.org" />
 	<target host="openbadges.es.python.org" />
+	<target host="socios.es.python.org" />
+	<target host="front.python.org" />
 	<target host="jobs.python.org" />
 	<target host="mail.python.org" />
-	<target host="packages.python.org" />
 	<target host="packaging.python.org" />
+	<target host="packages.python.org" />
 	<target host="pl.python.org" />
+	<target host="pypi.python.org" />
 	<target host="planet.python.org" />
 	<target host="speed.python.org" />
 	<target host="staging.python.org" />
+	<target host="staging-pycon.python.org" />
 	<target host="staging2.python.org" />
 	<target host="status.python.org" />
 	<target host="svn.python.org" />
-		<!-- 401 Authorization Required at root -->
-		<exclusion pattern="^http://svn\.python\.org/$" />
-		<test url="http://svn.python.org/projects/sftools/" />
 	<target host="testpypi.python.org" />
 	<target host="vote.python.org" />
 	<target host="warehouse.python.org" />
 	<target host="warehouse-staging.python.org" />
+	<target host="wiki.python.org" />
 
-	<securecookie host="^(bugs|cheeseshop|discuss|openbadges\.es|jobs|mail|packages|packaging|pl|planet|speed|staging|staging2|status|testpypi|warehouse|warehouse-staging|)\.python\.org$" name=".+" />
+	<securecookie host=".+" name=".+" />
 
-	<rule from="^http:"
-		to="https:" />
+	<rule from="^http:" to="https:" />
 
+	<!-- HTTP 403 at / -->
+	<exclusion pattern="^http://svn\.python\.org/$" />
```

Comment:
Please add a valid test url/keep the old one

Reply:
The test URL is already here, implicitly. Each target has an implicit test URL of `http://${domain}/`.

Reply:
Yes, but you just excluded this implicit test url. Now we have no tests for `svn.python.org` at all.

Reply:
@J0WI Done.